emerald-lang-spec/2num-operators.em

```diff
@@ -0,0 +1,32 @@
+ (+ 5 4)
+ ; compiles to:: 5 + 4
```

Comment:
Yes, I think that would be a nice way of verifying that it can compile longer lisp programs